osquery/dispatcher/scheduler.cpp

```diff
@@ -193,6 +194,12 @@ void SchedulerRunner::maybeRunDecorators(uint64_t time_step) {
   }
 }
 
+void SchedulerRunner::maybeScheduleCarves(uint64_t time_step) {
+  if ((time_step & 60) == 0) {
```

Comment:
Was this supposed to be `time_step % 60`?

Reply:
Good catch!